### ISSUE TITLE
Compliant documents

### ISSUE THREAD

**Issue by jpetersononrr** [ONRR/onrr.gov-site#387]
Hi!

I still need to work on the UCA's in our list, but here are the first 5 documents.  They passed the accessibility test on Adobe.  Please let me know if you need something else on these!  I left the file names the same so hopefully replacing is easy!
[20160913.pdf](https://github.com/ONRR/onrr.gov-site/files/5681179/20160913.pdf)left
[Uploading 20191017.pdf…]
[dear-reporter-letter-20-Nov-19.pdf](https://github.com/ONRR/onrr.gov-site/files/5681182/dear-reporter-letter-20-Nov-19.pdf)
[Dear-Reporter-Letter-2016-Rule.pdf](https://github.com/ONRR/onrr.gov-site/files/5681183/Dear-Reporter-Letter-2016-Rule.pdf)

Thanks!

[20141218.pdf](https://github.com/ONRR/onrr.gov-site/files/5681177/20141218.pdf)

**Comment by jpetersononrr** [comment]
Looks like one didn't attach correctly.
[20191017.pdf](https://github.com/ONRR/onrr.gov-site/files/5681572/20191017.pdf)

**Comment by Maroyafaied** [comment]
@jpetersononrr thank you for correcting these documents, we will double check for accessibility and update them if there are no errors.  @jennmalcolm please update the 508 spreadsheet once this issue is closed.

**Comment by jennmalcolm** [comment]
Good morning @jpetersononrr, when I ran the accessibility scans, I found that two documents still have a couple errors. I'm attaching those reports for you. Thanks.
[20160913.pdf.accreport.pdf](https://github.com/ONRR/onrr.gov-site/files/5688797/20160913.pdf.accreport.pdf)
[dear-reporter-letter-20-Nov-19.pdf.accreport.pdf](https://github.com/ONRR/onrr.gov-site/files/5688798/dear-reporter-letter-20-Nov-19.pdf.accreport.pdf)

**Comment by jennmalcolm** [comment]
I have replaced the three documents with no accessibility failures on the development and live sites.

**Comment by jpetersononrr** [comment]
Here's those 2 updated
[20160913.pdf](https://github.com/ONRR/onrr.gov-site/files/5751486/20160913.pdf)
[dear-reporter-letter-20-Nov-19.pdf](https://github.com/ONRR/onrr.gov-site/files/5751488/dear-reporter-letter-20-Nov-19.pdf)

**Comment by Maroyafaied** [comment]
@jpetersononrr Thank you, I ran an accessibility check for both documents and there were no errors. @jennmalcolm will post these next week when she's back from leave. Thanks!

**Comment by jpetersononrr** [comment]
Splendid!  I have these 3 ready as well.  They all came back with green checks when I ran the accessibility checker on Adobe.  I only owe you one more I believe.  Thanks!

[Ignacio UCA Table 20180208.pdf](https://github.com/ONRR/onrr.gov-site/files/5755002/Ignacio.UCA.Table.20180208.pdf) 
[San Juan Transportation System - Chaco Gas Plant.pdf](https://github.com/ONRR/onrr.gov-site/files/5755003/San.Juan.Transportation.System.-.Chaco.Gas.Plant.pdf)
[DCP Super system UCA.pdf](https://github.com/ONRR/onrr.gov-site/files/5755001/DCP.Super.system.UCA.pdf)

**Comment by Maroyafaied** [comment]
@jpetersononrr looks like the San Juan doc has one error, I've attached the report. The other two are good. Thanks
[San.Juan.Transportation.System.-.Chaco.Gas.Plant accessibility report.pdf](https://github.com/ONRR/onrr.gov-site/files/5757700/San.Juan.Transportation.System.-.Chaco.Gas.Plant.accessibility.report.pdf)

**Comment by jpetersononrr** [comment]
NOOOOOOOO!!!  I went over that one tons and got it to pass.  The regularity thing I can’t reconcile, but I had magically at one point.  The rows and columns are what they are.  I don’t know why Adobe doesn’t like them…I can’t.  I’ve messed with it all the way to the point of deleting everything in the document and it still won’t pass.  If you have a way, feel free!

​Jodie Peterson
Minerals Revenue Specialist
Office of Natural Resources Revenue
​jodie.peterson@onrr.gov
(303)231-3719

Royalty Valuation - Consistent Answers to Complex Questions

**Comment by Maroyafaied** [comment]
@jpetersononrr , Jennifer will look into that document and see if she has a way to correct it. @jennmalcolm this is the other document that Jodi is having a hard time correcting and I tried as well, the issue is from the brackets used in the math equations, if you can also look into it to see if there's a way around it that would be great. Thanks
[How to calculate a Transportation UCA.pdf](https://github.com/ONRR/onrr.gov-site/files/5766118/How.to.calculate.a.Transportation.UCA.pdf)

**Comment by jennmalcolm** [comment]
I've replaced all of the documents. I wasn't able to fix the font issue or the table issues on page three, but I did fix the table issues on page 1-2 in the Chaco file.
